api/test/queries.test.js

```diff
@@ -20,6 +20,351 @@ describe('queries', () => {
+          {
+            creationDate: '2012-10-01T15:08:39',
+            entryDate: '2011-08-18',
+            evaluationType: 'D',
+            modificationDate: '2012-06-09T11:20:20',
+          },
+        ],
+      })
+    })
+
+    it('retrieves all top level keys of the ceiling data', async () => {
+      let response = await request(server)
+        .post('/graphql')
+        .set('Content-Type', 'application/json; charset=utf-8')
+        .send({
+          query: `{
+          evaluations:evaluationsFor(account: 189250) {
+            evaluations {
+              ceilings {
+                label
+                typeEnglish
+                typeFrench
+                nominalRsi
+                nominalR
+                effectiveRsi
+                effectiveR
+                areaMetres
+                areaFeet
+                lengthMetres
+                lengthFeet
+              }
+            }
+          }
+        }`,
+        })
+
+      let { evaluations } = response.body.data
+      expect(evaluations).toEqual({
+        evaluations: [
+          {
+            ceilings: [
+              {
+                areaFeet: 830.1171300283143,
+                areaMetres: 77.1204,
+                effectiveR: 36.3397497041326,
+                effectiveRsi: 6.3998,
+                label: 'Ceiling01',
+                lengthFeet: 47.66404352,
+                lengthMetres: 14.528,
+                nominalR: 36.3397497041326,
+                nominalRsi: 6.3998,
+                typeEnglish: 'Attic/gable',
+                typeFrench: 'Combles/pignon',
+              },
+            ],
+          },
+          {
+            ceilings: [
+              {
+                areaFeet: 830.1171300283143,
+                areaMetres: 77.1204,
+                effectiveR: 36.3397497041326,
+                effectiveRsi: 6.3998,
+                label: 'Ceiling01',
+                lengthFeet: 47.66404352,
+                lengthMetres: 14.528,
+                nominalR: 36.3397497041326,
+                nominalRsi: 6.3998,
+                typeEnglish: 'Attic/gable',
+                typeFrench: 'Combles/pignon',
+              },
+            ],
+          },
+        ],
+      })
+    })
+
+    it('retrieves all top level keys of the wall data', async () => {
+      let response = await request(server)
+        .post('/graphql')
+        .set('Content-Type', 'application/json; charset=utf-8')
+        .send({
+          query: `{
+          evaluations:evaluationsFor(account: 189250) {
+            evaluations {
+              walls {
+                label
+                structureTypeEnglish
+                structureTypeFrench
+                componentTypeSizeEnglish
+                componentTypeSizeFrench
+                nominalRsi
+                nominalR
+                effectiveRsi
+                effectiveR
+                areaMetres
+                areaFeet
+                perimeter
+                height
```

Comment:
I think you want `perimeterMetres`, `perimeterFeet `, `heightMetres` and `heightFeet` here